app/models/apple/config.rb

```diff
@@ -57,6 +57,7 @@ def self.find_or_build_private_feed(podcast)
         audio_format: DEFAULT_AUDIO_FORMAT,
         include_zones: ["billboard", "sonic_id"],
         tokens: [FeedToken.new(label: DEFAULT_TITLE)],
+        uncached: true,
```

Comment:
What about a cache timeout value instead of a boolean? would that ever be needed - to set a cache value that could be nothing, or some other value? hmm, maybe more complicated for no good reason, this is good.